tiledb/bioimg/converters/base.py

```diff
@@ -353,6 +398,31 @@ def _convert_level_to_tiledb(
         compressor.get(level, tiledb.ZstdFilter(level=0)),
     )
 
+    min_max_indices = tuple(
+        idx for idx, char in enumerate(source_axes.dims) if char != "C"
+    )
+
+    channel_index = source_axes.dims.find("C")
+    channel_count = source_shape[channel_index] if channel_index > -1 else 1
```

Comment:
Could you comment your code in places with small comments to explain what happens.